setup.py

```diff
@@ -92,7 +92,7 @@ def emit(self, record):
                     _unittests_only(suite, mod.test_suite())
     return suite
 
-tests_require = ['zope.testing', 'manuel', 'random2']
+tests_require = ['zope.testing', 'manuel', 'random2', 'mock']
```

Comment:
I hope not. :) Dependencies should be fully specified in the setup.py.  If Tox required restating dependencies, that would be a strike against Tox.  I don't use Tox myself.  I'll use this as an excuse to try it and make sure it works.